index.html

```diff
@@ -34,6 +34,7 @@
  *   - .note       for informative notes             (div, p, span, aside, details)
  *   - .example    for informative examples          (div, p, pre, span)
  *   - .issue      for issues                        (div, p, span)
+ *   - .assertion  for assertions                    (div, p, span)
```

Comment:
it's hard to follow all these changes. Have we changed the generator at the same time and it has resulted in all these changes?

Reply:
You should ignore the index.html diff as it is the reflection of index.bs + dependencies. Same reasoning as I said in #69